config/nginx/django_https.ctmpl

```diff
@@ -10,7 +10,7 @@ server {
 
     server_name www.devinit.org devinit.org;
 
-    client_max_body_size 10M;
+    client_max_body_size 100M;
```

Comment:
I added this ... it's supposed to fix [this](https://github.com/devinit/DIwebsite-redesign/issues/793#issuecomment-671249922) issue